fortran-inference/inference_scaling_imagenet.F90

```diff
@@ -0,0 +1,252 @@
+program main
```

Comment:
The output times in the fortran test have spacing that look like `2,put_tensor,   0.6258940696716309E-01` -> the cpp csv lines look like `0,unpack_tensor,0.0587351`. Im wondering if the space between the function name and time for the fortran csv code will affect the performance results parse.

Reply:
client() and main() do not have these spaces in fortran csv files